src/utils/claimgraph.js

```diff
@@ -69,7 +69,7 @@ export function fromJsonldjsNode(jn) {
 // Blank node name generator
 // An object of this type won't produce the same name twice, even accross multiple calls to
 // reallocateNames.
-class Namer {
+export class Namer {
```

Comment:
@SamHellawell is there a good way to expose this to other parts of the sdk without exposing it to users?

Reply:
not really, by the nature of JS if its exported its publicly available through importing that file and that the SDK separates files. If we were to bundle a bunch of the files then sure, maybe it could be "hidden". Still, I don't think its a big concern. If you don't want to expose it in utils/claimgraph put it in another file and import it into claimgraph. That file will still be available for the user to import directly from if they want though.

Reply:
Thanks